src/main/java/pl/put/poznan/JSONtools/logic/JSONDecorator.java

```diff
@@ -0,0 +1,17 @@
+package pl.put.poznan.JSONtools.logic;
```

Comment:
What branch did you create this branch from? These files (this+minifier+prettifier) should not be here yet. Please create all branches from the main branch, to avoid possibly overwriting some bug fixes with broken versions of code.